source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,68 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``Collection.UpdateOne()``
+method. 
+
+This example passes a query filter and an update parameter to the
+``UpdateOne()`` method. The query filter specifies the document to
+match, while the update parameter specifies the change to
+make to the document. The following code example calls the
+``UpdateOne()`` method on a document matched by its ``_id`` field from
+the ``restaurants`` collection to create a new field called
+``avg_rating`` with a value of 4.4:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateOne.go
+   :start-after: begin updateone
+   :end-before: end updateone
+   :emphasize-lines: 6
+   :language: go
+   :dedent:
+
+Click here <TODO: github link to file> to see a fully runnable example.
+
+.. note::
+
+   If the query filter does not match any documents, the operation will
+   return a result with a ``MatchedCount`` property of 0. If the filter
+   matches multiple documents, the method will select and update one document.
```

Comment:
question/comment: Shouldn't this be in the Fundamental page?

Reply:
accepted